### DISCUSSION TITLE
SyliusAdminApiBundle on Sylius 1.10

### DISCUSSION THREAD

**Discussion by kg4b0r** [Sylius/Sylius#13007] [Q&A] [answered]
Hi,

The new version of the API integrated in 1.10 is missing a lot of essential endpoints (PATCH mainly) and the roadmap shows these are coming only in 2022. 

Is there a way to use SyliusAdminApiBundle on a fresh Sylius 1.10 install?

**Comment by CoderMaggie** [comment, marked as answer]
Sylius new, full API had already been released 🫡
